pkg/shares/namespaced_padding.go

```diff
@@ -33,3 +39,16 @@ func NamespacedPaddedShares(ns namespace.ID, n int) []Share {
 	}
 	return shares
 }
+
+func IsNamespacedPadded(s Share) (bool, error) {
+	isSequenceStart, err := s.IsSequenceStart()
+	if err != nil {
+		return false, err
+	}
+	sequenceLen, err := s.SequenceLen()
+	if err != nil {
+		return false, err
+	}
+
+	return isSequenceStart && sequenceLen == 0, nil
```

Comment:
[Question][Non-blocking]
Is there another case where this condition is true but it's not a namespaced padded? For example posting an empty blob?

Reply:
> For example posting an empty blob?

empty blobs are invalid, so this is something that we have to (and are) checking for https://github.com/celestiaorg/celestia-app/blob/580d1864f7179689e1aa9994dbb526115f1a175e/x/blob/types/payforblob.go#L208-L210

that func is called when checking a tx before its accepted in the mempool, and before creating a PFB

Reply:
Cool, thanks :+1: :+1: